src/status_im/ui/screens/privacy_and_security_settings/views.cljs

```diff
@@ -12,20 +12,21 @@
   (:require-macros [status-im.utils.views :as views]))
 
 (defn- list-data [show-backup-seed? settings]
-  [{:type                 :section-header
-    :title                :t/security
-    :container-margin-top 6}
-   {:type                    :small
-    :title                   :t/back-up-seed-phrase
-    :accessibility-label     :back-up-recovery-phrase-button
+  [(when show-backup-seed?
+     {:type                 :section-header
+      :title                :t/security
+      :container-margin-top 6}
+     {:type                    :small
+      :title                   :t/back-up-seed-phrase
+      :accessibility-label     :back-up-recovery-phrase-button
     ;; TODO - remove container bottom margin
     ;; when items below are implemented
-    :container-margin-bottom 8
-    :on-press
-    #(re-frame/dispatch [:navigate-to :backup-seed])
-    :accessories
-    [(when show-backup-seed? [components.common/counter {:size 22} 1])
-     :chevron]}
+      :container-margin-bottom 8
+      :on-press
+      #(re-frame/dispatch [:navigate-to :backup-seed])
+      :accessories
+      [(when show-backup-seed? [components.common/counter {:size 22} 1])
```

Comment:
we don't need this show-backup-seed? condition then

Reply:
@flexsurfer  Removed the condition.  Secondary question not related to the main PR, what's the point of the counter?  It doesn't appear to be rendering in the UI.

Reply:
@andmironov ?

Reply:
Nevermind, I figured it out.  It only shows when you haven't actually backed up your seedphrase (as in the design below from @andmironov).  Just me being clueless.